### ISSUE TITLE
Tolerate missing `;` at the end of structure definitions

### ISSUE THREAD

**Issue by op2786** [Vector35/binaryninja-api#3849]
I frequently forget to put a `;` at the end of structure /enum /union definitions. Can BN tolerate that?

```c
struct SERVICE_DESCRIPTIONA
{
    char* lpDescription;
}
```

```
error: sources.hpp:4:2 expected ';' after struct
1 error generated.
```

**Comment by plafosse** [comment]
This can probably be done when its a single structure definition. Glenn would know better.

**Comment by CouleeApps** [comment]
This is likely easy to do. Someone just needs to look into if appending a semicolon can ever cause a valid string to no longer be valid.

**Comment by fuzyll** [comment]
I have modified the UI to silently append a `;` to the entire contents of the box. This is either a no-op, or fixes this issue for single `struct`/`enum` definitions.

I didn't test this _exhaustively_, but nothing I tested broke as a result of this change. Having multiple definitions and forgetting the semi-colon still doesn't work, but no one indicated that was a requirement of this issue and the behavior is still basically unchanged (except that the last one will now work). So, I'm closing it as fixed.

If anyone has an input that this change _breaks_, please comment and I will do my best to address any regressions I've introduced.

This new behavior is available in builds >= 3.5.4322.
